### PR TITLE
Add Typography Component Group

### PR DESCRIPTION
**Pull request by GideonJesugbemi**
## Description

Typography Component Group:

This component group is designed to leverage the @tailwindcss/typography plugin, enhancing the typographic capabilities of our project. The group will consist of the following components:

Title (new component)
TypographyWrapper (new component)

## Related Issue

- #108 

## Contribution Guidelines

Before submitting this pull request, please review our [Contribution Guidelines](https://github.com/abeg-help/frontend/blob/dev/CONTRIBUTING.md) to understand how to contribute to this project.

## Checklist

- I have reviewed the Contribution Guidelines linked above.
-  I have tested my changes thoroughly and ensured that all existing tests pass.
-  I have provided clear and concise commit messages.
-  I have updated the project's documentation as necessary.

## Screenshots (if applicable)

TypographyWrapper
Title

### CODE REVIEW

File: primitives/TypographyWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from 'react';
```
Comment (babblebey) [inline primitives/TypographyWrapper.tsx]:
```suggestion
import React, { type ReactNode } from 'react';
```

When importing types, let's take the practice of prepending the `type` keyword to it, so to give cue of what it is.

File: primitives/Title.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react';
```
Comment (babblebey) [inline primitives/Title.tsx]:
```suggestion
import React, { type ReactNode } from 'react';
```

Explicitly stating that the import is a `type`

File: primitives/TypographyWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+interface Iprops {
```
Comment (babblebey) [inline primitives/TypographyWrapper.tsx]: Do rename this interface to match the component name

File: primitives/TypographyWrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  className?: string;
+  size?: string;
```
Comment (babblebey) [inline primitives/TypographyWrapper.tsx]: Something is missing here 😉

The various sizes `sm | base | lg` etc...

**Review by babblebey — CHANGES_REQUESTED** [review]
Great stuff @GideonJesugbemi 

Kindly address the changes below.

...and yo, I seriously look forward to a demo of this 😉, this is because I expected a configuration of the `@tailwind/typography` plugin which should trigger a change in the `tailwind.config.ts` file.

So, I'm just super curious to know whether this component work without that configuration or did you forget to commit the config file???. 🤘🏾

**Review by 289Volts — APPROVED** [review]
Good job on this one

**Pull request merged**